adobe-eventbridge-lambda/adobe-eventbridge-lambda/adobe-eventbridge-lambda.json

```diff
@@ -0,0 +1,90 @@
+{
+    "title": "Adobe EventBridge Integration to AWS Lambda",
+    "description": "Use the Amazon EventBridge Adobe I/O Events integration to process/route order events with AWS Lambda, and notify users with Amazon SES or store the data elsewhere.",
```

Comment:
The description can have at most 150 characters, please reword it to fit the limit